aptos-move/move-examples/rewards_pool/simple/sources/rewards.move

```diff
@@ -0,0 +1,82 @@
+module rewards::rewards {
```

Comment:
@kshitjee  Add a function transfer_admin to transfer ownership of code object (ability to upgrade) + write a short tutorial and add to README file